documenters_aggregator/spiders/ccc.py

```diff
@@ -0,0 +1,139 @@
+from pytz import timezone
+
+
+class CccSpider(scrapy.Spider):
+    name = 'ccc'
+    allowed_domains = ['http://www.ccc.edu/departments/Pages/Board-of-Trustees.aspx']
+    start_urls = ['http://http://www.ccc.edu/departments/Pages/Board-of-Trustees.aspx']
+
+    # for each link
+    #   parse 
+    def parse(self, response):
+        """
+        `parse` should always `yield` a dict that follows the `Open Civic Data
+        event standard <http://docs.opencivicdata.org/en/latest/data/event.html>`_.
+
+        Change the `_parse_id`, `_parse_name`, etc methods to fit your scraping
+        needs.
+        """
+        for link in response.css('a.Eventslink::attr(href)').extract():
+            next_url = "http://www.ccc.edu" + link
+            return scrapy.Request(next_url, callback=self.parse_event_page())
+
+    def parse_event_page(self, response):
+            yield {
+                '_type': 'event',
+                'id': self._parse_id(),
+                'name': self._parse_name(),
+                'description': self._parse_description(),
+                'classification': self._parse_classification(),
+                'start_time': self._parse_start(),
+                'end_time': self._parse_end(),
+                'all_day': self._parse_all_day(),
+                'status': self._parse_status(),
+                'location': self._parse_location(),
+            }
+
+        # self._parse_next(response) yields more responses to parse if necessary.
+        # uncomment to find a "next" url
+        # yield self._parse_next(response)
+
+    def _parse_next(self, response):
+        """
+        Get next page. You must add logic to `next_url` and
+        return a scrapy request.
+        """
+        next_url = None  # What is next URL?
+        return None
+
+    def _parse_id(self, item):
+        """
+        Calulate ID. ID must be unique within the data source being scraped.
+        """
+        return None
+
+    def _parse_classification(self, item):
+        """
+        Parse or generate classification (e.g. town hall).
+        """
+        return 'Not classified'
+
+    def _parse_status(self, item):
+        """
+        Parse or generate status of meeting. Can be one of:
+
+        * cancelled
+        * tentative
+        * confirmed
+        * passed
+
+        By default, return "tentative"
+        """
+        return 'tentative'
+
+    def _parse_location(self, item):
+        """
+        Parse or generate location. Url, latitutde and longitude are all
+        optional and may be more trouble than they're worth to collect.
+        """
+        return {
+            'url': None,
+            'name': None,
+            'coordinates': {
+              'latitude': None,
+              'longitude': None,
+            },
+        }
+
+    def _parse_all_day(self, item):
+        """
+        Parse or generate all-day status. Defaults to false.
+        """
+        return False
+
+    def _parse_name(self, item):
```

Comment:
This is fine, but for consistency/ease of understanding, it might be better to do `def _parse_name(self, response)`. Same for _parse_description, _parse_start, etc.